### ISSUE TITLE
released checkpoint: veri_871.pth's results

### ISSUE THREAD

**Issue by jiaohere** [cybercore-co-ltd/track2_aicity_2021#5]
Hi,
   Thank you for your released code! I have some questions when testing with the released veri_871.pth.
1. When tested without post-processing technique, I got mAP=78.7%;
2. I can get the reported mAP 87.1% when using re-rank technique. But I thought it was clarified in the paper that non post-processing was used.
    "For a fair comparison, we only use single model, including backbone ResNeXt101 ibn a, multihead, CE and SupCon losses and MALW without applying pos-processing technique and synthetic data. We achieve the state-of-the-art performance with a large margin compared to previous works" (sec 4.4)
3. Also I have reproduced the VOC-ReID repo: res50_ibn_a, which got the results on veri-776:  **81.5% (w/o re-rank);  87.5% (with re-rank)**. It seems that larger backbone / multihead / MALW do not brought bonus here?
    If I have any misunderstandings, please help me to point out. Thank you very much. Attached are the testing logs.
[voc_reid_test_with_rerank.txt](https://github.com/cybercore-co-ltd/track2_aicity_2021/files/6798981/voc_reid_test_with_rerank.txt)
[veri_871_test_with_rerank.txt](https://github.com/cybercore-co-ltd/track2_aicity_2021/files/6798985/veri_871_test_with_rerank.txt)

**Comment by llf1234** [comment]
hi how to test veri_871.pth?  how to write veri.yml and test.sh 。the network of  veri_871.pth is ResNeXt101 ibn a？？

**Comment by jiaohere** [comment]
> hi how to test veri_871.pth? how to write veri.yml and test.sh 。the network of veri_871.pth is ResNeXt101 ibn a？？

Yes, you can also find detail configs in my attached log file: veri_871_test_with_rerank.txt

**Comment by llf1234** [comment]
thank you ！ if we not use rerank，map is 78.7%

**Comment by ra4-z** [comment]
> > hi how to test veri_871.pth? how to write veri.yml and test.sh 。the network of veri_871.pth is ResNeXt101 ibn a？？
> 
> Yes, you can also find detail configs in my attached log file: veri_871_test_with_rerank.txt

Sorry but do you still have the .pth file? I want it but it seems gone rn.
